File: Desafios em C#/2717 - Tempo do Duende.cs
using System;

namespace _2717___Tempo_do_Duende
{
    class URI {
    
        public static void Main (string[] args) {
        
            int finalExpediente = int.Parse(Console.ReadLine());
            
            string[] linha_min = Console.ReadLine().Split(' ');
            
            int presente1 = int.Parse(linha_min[0]);
            int presente2 = int.Parse(linha_min[1]);
            
            int total =  presente1 + presente2;
            
            if (total > finalExpediente)
            {
            Console.WriteLine("Deixa para amanha!");
            }
            else
            {
            Console.WriteLine("Farei hoje!");
            }
        }
    }
}

File: Desafios em JavaScript/2717 - Tempo do Duende.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var finalExpediente = parseInt(lines.shift());

var ln = lines.shift().split(` `);

var presente1 = parseInt(ln.shift());
var presente2 = parseInt(ln.shift());

var total = presente1 + presente2;

if (total > finalExpediente)
{
    console.log("Deixa para amanha!");
}
else
{
    console.log("Farei hoje!");
}